### PR TITLE
match makecode log

### PR DESCRIPTION
**Pull request by pelikhan**
- [x] match log categories in makecode.
- [x] use properties for minimumPriority

### CODE REVIEW

File: source/consoleservice.ts [hunk 1]
```diff
@@ -29,7 +38,7 @@ namespace jacdac {
             if (this.mode == JDServiceMode.BroadcastHostService) {
                 let consolePkt = new JDConsolePacket(pkt);
 
-                if (consolePkt.priority < ((this.status & 0xF0) >> 4))
+                if (consolePkt.priority < this.minimumPriority)
```
Comment (jamesadevine) [inline source/consoleservice.ts]: There's no minimum priority property on the class

File: source/consoleservice.ts [hunk 2]
```diff
@@ -50,15 +60,19 @@ namespace jacdac {
             return DEVICE_OK;
         }
 
-        setMinimumPriority(priority: JDConsoleLogPriority) {
+        get minimumPriority() {
```
Comment (pelikhan) [inline source/consoleservice.ts]: this is the minimumProperty (``get`` keyword)

File: source/consoleservice.ts (hunk 1 above)
Comment (pelikhan) [reply source/consoleservice.ts]: see "get minimumProperty()"

Comment (jamesadevine) [reply source/consoleservice.ts]: Ah, missed that

**Comment by jamesadevine** [comment]
LGTM

**Pull request merged**